Commons/src/main/java/com/github/jsonldjava/core/JsonLdApi.java

```diff
@@ -365,14 +374,15 @@ public Object compact(Context activeCtx, String activeProperty, Object element,
 						} else {
 							continue;
 						}
-					} else if (isLanguageProperty) {
-						if (expandedProperty.equals(NGSIConstants.NGSI_LD_HAS_LANGUAGE_MAP)) {
-							return compact(activeCtx, activeProperty, expandedValue, compactArrays, endPoint, null,
-									null);
-						} else {
-							continue;
-						}
 					}
+//					else if (isLanguageProperty) {
```

Comment:
Remove unused code..

Reply:
Left for future purposes in case something goes wrong